app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -88,9 +113,11 @@ export default function ResourceAuthoringPage() {
         </Grid.Col>
         <Grid.Col span={6}>
           <Paper withBorder>
-            <Prism language="json" colorScheme="light">
-              {resourceQuery.data ? JSON.stringify(resourceQuery.data, null, 2) : ''}
-            </Prism>
+            <div className={classes.json}>
+              <Prism language="json" colorScheme="light">
```

Comment:
It makes more sense here to use the [styles API](https://mantine.dev/others/prism/?t=styles-api) to customize the default scroll area container of the `Prism` rather than wrap it in a div. With this div, we need to scroll all the way to the bottom to get a horizontal scrollbar, and the copy button disappears. Something like this should work better:

```
              <Prism
                styles={{
                  scrollArea: {
                    height: 'calc(100vh - 150px)'
                  }
                 // ...
                }}
              >
```

And remove the custom `json` class from the parent